Misc/NEWS.d/next/Tests/2020-06-23-12-02-45.bpo-41085.JZKsyz.rst

```diff
@@ -0,0 +1 @@
+Fixed 'array_array_index' downcasting 'Py_ssize_t' to 'long'
```

Comment:
(copy/paste of my comment :-))     

I suggest to write a more high-level description for end users, like:

"Fix integer overflow in the array.array.index() method on 64-bit Windows for index larger than 2**31."